packages/examples/packages/lifecycle-hooks/src/index.tsx

```diff
@@ -85,3 +86,43 @@ export const onUpdate: OnUpdateHandler = async () => {
     },
   });
 };
+
+/**
+ * Handle activation of the client. This handler is called when the client is
+ * activated, and can be used to perform any initialization that is required.
+ *
+ * This handler is optional.
+ *
+ * @see https://docs.metamask.io/snaps/reference/entry-points/#onactive
+ * @returns The JSON-RPC response.
+ */
+export const onActive: OnActiveHandler = async () => {
+  return await snap.request({
+    method: 'snap_notify',
```

Comment:
Do we have notifications on the mobile side ? I'm worried that we will not be able to introduce a E2E if it's not available

Reply:
Doesn't seem to be implemented 😕

https://github.com/MetaMask/metamask-mobile/blob/60361c21a80fe101062afa3d2e9eb668767b1e21/app/core/Engine/Engine.ts#L780-L787

Do you have a better suggestion? I don't think dialogs would work well here either.